universal-ctags.rb

```diff
@@ -1,17 +1,24 @@
 class UniversalCtags < Formula
   homepage 'https://github.com/universal-ctags/ctags'
   head 'https://github.com/universal-ctags/ctags.git'
+
+  option "without-iconv", "Enable multibyte character encoding"
```

Comment:
In `homebrew` defining an option as `without-something` enables that option by default, and offers the options to disable it when installing by adding `--without-something` to the `brew install` command. So my change actually enables `iconv` by default, changing the previous behaviour that didn't configure it.